File: benchmarks/corpus/csharp/SqlInjection.cs
using System.Data.SqlClient;
using Microsoft.AspNetCore.Mvc;

public class UsersController : Controller
{
    public IActionResult Lookup(string name)
    {
        var conn = new SqlConnection(ConnStr);
        var cmd = new SqlCommand("SELECT * FROM users WHERE name = '" + name + "'", conn);
        cmd.ExecuteReader();
        return Ok();
    }
}

File: benchmarks/corpus/java/SqlInjection.java
package com.example;

import java.sql.Connection;
import java.sql.Statement;
import javax.servlet.http.HttpServletRequest;

public class SqlInjection {
    public void lookup(Connection conn, HttpServletRequest request) throws Exception {
        String name = request.getParameter("name");
        Statement stmt = conn.createStatement();
        stmt.executeQuery("SELECT * FROM users WHERE name = '" + name + "'");
    }
}
